NEWS.md

```diff
@@ -9,6 +9,7 @@ We aim at 3 to 4 month between major release versions and about 2 weeks between
 ### Highlights
 
 ### Added
+Particle sampling for complex shapes from `.stl` and `.asc` files.
 
 ### Removed
 

```

Comment:
This is not the correct format for the current dev cycle i.e.

## Version 0.2.1
### Added
Particle sampling for complex shapes from `.stl` and `.asc` files.